randovania/gui/ui_files/games_tab_am2r_widget.ui

```diff
@@ -156,6 +156,414 @@
     </item>
    </layout>
   </widget>
+  <widget class="QWidget" name="hints_tab">
+   <attribute name="title">
+    <string>Hints</string>
+   </attribute>
+   <layout class="QVBoxLayout" name="hints_tab_layout_4">
+    <property name="spacing">
+     <number>0</number>
+    </property>
+    <property name="leftMargin">
+     <number>0</number>
+    </property>
+    <property name="topMargin">
+     <number>0</number>
+    </property>
+    <property name="rightMargin">
+     <number>0</number>
+    </property>
+    <property name="bottomMargin">
+     <number>0</number>
+    </property>
+    <item>
+     <widget class="QScrollArea" name="hints_scroll_area">
+      <property name="sizePolicy">
+       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+        <horstretch>0</horstretch>
+        <verstretch>0</verstretch>
+       </sizepolicy>
+      </property>
+      <property name="widgetResizable">
+       <bool>true</bool>
+      </property>
+      <widget class="QWidget" name="hints_scroll_area_contents">
+       <property name="geometry">
+        <rect>
+         <x>0</x>
+         <y>0</y>
+         <width>755</width>
+         <height>630</height>
+        </rect>
+       </property>
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <layout class="QVBoxLayout" name="hints_scroll_layout_4">
+        <property name="spacing">
+         <number>6</number>
+        </property>
+        <property name="topMargin">
+         <number>6</number>
+        </property>
+        <property name="rightMargin">
+         <number>6</number>
+        </property>
+        <property name="bottomMargin">
+         <number>6</number>
+        </property>
+        <item>
+         <widget class="QLabel" name="hints_label">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="text">
+           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In AM2R, you can find hints from the following sources: &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Broken Chozo Statue&lt;/span&gt;: Hints for where your Ice Beam is located. It can be found in Genetics Laboratory - Destroyed Chozo Memorial. &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Wisdom Septoggs&lt;/span&gt;: Wisdom Septoggs have been placed throughout the planet, one in each Area. They hint to where your Metroid DNA is located. Each location can hint between zero DNA and a specific maximum. If a Wisdom Septogg hints zero DNA, it will display a joke hint instead. The locations and maximum amount of hinted DNA is as follows:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
```suggestion
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In AM2R, you can find hints from the following sources: &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Broken Chozo Statue&lt;/span&gt;: Hints for where your Ice Beam is located. It can be found in Genetics Laboratory - Destroyed Chozo Memorial. &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Wisdom Septoggs&lt;/span&gt;: Wisdom Septoggs have been placed throughout the planet, one in each Area. They hint to where your Metroid DNA is located. Each Wisdom Septogg can hint between zero DNA and a set maximum. If no hunts are provided, a joke hint is displayed instead. The locations and maximum amount of hinted DNA is as follows:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```